frontend/store/auctions.ts

```diff
@@ -54,11 +54,11 @@ const getInitialState = (): State => ({
 export const state = (): State => getInitialState();
 
 export const getters = {
-    listAuctions(state: State): AuctionTransaction[] {
-        return Object.values(state.auctionStorage);
-    },
-    listAuctionTransactions(state: State, getters: any, _rootState: any): AuctionTransaction[] {
-        const auctions = Object.values(state.auctionStorage);
+    // `rootGetters` is exposed to the `getters` module as the 4th parameter, right after `rootState`
+    // Hence the dummy _rootState parameter
```

Comment:
If you want to leave comment to the reviewer, you can make a comment in the PR itself, not in the code 🙂 
```suggestion
```